mne/tests/test_docstring_parameters.py

```diff
@@ -62,7 +62,8 @@ def get_name(func):
     'mne.utils.plot_epochs_trellis',  # deprecated
     'mne.utils.write_bem_surface',  # deprecated
     'mne.gui.coregistration',  # deprecated
-    'mne.utils.plot_topo'
+    'mne.utils.plot_topo',
+    'mne.viz.plot_image_epochs'  # deprecated
```

Comment:
FYI if you throw a comment at the end of this line, then the next diff that adds something to this will be a tiny bit easier to read (no need to do it here though)